runtime/include/chpl-mem-sys.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2004-2017 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _chpl_mem_sys_H_
+#define _chpl_mem_sys_H_
+
+// disable mem warnings since we need to call the system allocator
+#include "chpl-mem-no-warning-macros.h"
```

Comment:
Since Elliot is planning to get rid of these warning macros files in a subsequent PR I think it makes more sense to leave them as-is for now. That way he can get more glory out of the next PR :)